geotools-suite/pom.xml

```diff
@@ -134,28 +134,158 @@
+							org.geotools.referencing.operation;version=${bundle.version},
+							org.geotools.referencing.operation.builder;version=${bundle.version},
+							org.geotools.referencing.operation.matrix;version=${bundle.version},
+							org.geotools.referencing.operation.projection;version=${bundle.version},
+							org.geotools.referencing.operation.transform;version=${bundle.version},
+							org.geotools.referencing.wkt;version=${bundle.version},
+							org.geotools.resources;version=${bundle.version};-split-package:=merge-first,
+							org.geotools.resources.geometry;version=${bundle.version},
+
+							<!-- gt-jts-wrapper -->
+							org.geotools.geometry.jts;version=${bundle.version},
+							org.geotools.geometry.jts.spatialschema;version=${bundle.version},
+							org.geotools.geometry.jts.spatialschema.geometry;version=${bundle.version},
+							org.geotools.geometry.jts.spatialschema.geometry.aggregate;version=${bundle.version},
+							org.geotools.geometry.jts.spatialschema.geometry.complex;version=${bundle.version},
+							org.geotools.geometry.jts.spatialschema.geometry.geometry;version=${bundle.version},
+							org.geotools.geometry.jts.spatialschema.geometry.primitive;version=${bundle.version},
+
+							<!-- gt-xsd-filter -->
+							org.geotools.xml.filter;version=${bundle.version},
+							org.geotools.filter.v1_0;version=${bundle.version},
+							org.geotools.filter.v2_0;version=${bundle.version},
+							org.geotools.filter.v1_1;version=${bundle.version},
+
+							<!-- gt-xsd-gml2 -->
+							org.geotools.gml2;version=${bundle.version},
+							org.geotools.gml2.bindings;version=${bundle.version},
+
+							<!-- gt-xsd-gml3 -->
+							org.geotools.gml3;version=${bundle.version},
+							org.geotools.gml3.bindings;version=${bundle.version},
+							org.geotools.gml3.bindings.ext;version=${bundle.version},
+							org.geotools.gml3.smil;version=${bundle.version},
+							org.geotools.gml3.v3_2;version=${bundle.version},
+							org.geotools.gml3.v3_2.bindings;version=${bundle.version},
+							org.geotools.gml3.v3_2.gco;version=${bundle.version},
+							org.geotools.gml3.v3_2.gmd;version=${bundle.version},
+							org.geotools.gml3.v3_2.gmx;version=${bundle.version},
+							org.geotools.gml3.v3_2.gsr;version=${bundle.version},
+							org.geotools.gml3.v3_2.gss;version=${bundle.version},
+							org.geotools.gml3.v3_2.gts;version=${bundle.version},
+
+							<!-- gt-xsd-core -->
+							org.geotools.xml;version=${bundle.version};-split-package:=merge-first,
+							org.geotools.xml.schema;version=${bundle.version};-split-package:=merge-first,
+							org.geotools.xml.impl;version=${bundle.version};-split-package:=merge-first,
+							org.geotools.xml.handlers.xsi;version=${bundle.version};-split-package:=merge-first,
+							org.geotools.xlink;version=${bundle.version},
+							org.geotools.xs;version=${bundle.version},
+							org.geotools.xs.bindings;version=${bundle.version},
+
+							<!-- gt-wfs -->
```

Comment:
❓ I'm just guessing here based on the section comment...should these three versions be set to `16.5` instead of `${bundle.version}`?

Reply:
I'm not sure. Their WFS stuff has been discontinued and I was trying to just keep everything at the same level. I can change it if folks think it should be changed, but then it might complicate how the dependencies are pulled in on the DDF side. Might seem weird for a 19.x bundle exporting 16.x versions. 🤷‍♂️